assets/javascripts/discourse/components/minimal-gamification-leaderboard.js

```diff
@@ -30,6 +30,9 @@ export default class extends Component {
       if (user.id === this.model.personal?.user?.id) {
         user.isCurrentUser = "true";
       }
+      if (user.id === 1) {
```

Comment:
id? 😃

Reply:
it looked weird to me too, prolly didnt understand it fully… what should I use then?

Reply:
ok wait I know I think

Reply:
K, nvm I dont